documentation/modules/ROOT/pages/configuration/applying-transformations-selectively.adoc

```diff
@@ -0,0 +1,190 @@
+
+The situation is similar for predicates that you define for a sink connector SMT.
+The connector reads messages from a Kafka topic and Kafka Connect evaluates the messages against the predicate condition.
+If a message matches the condition, Kafka Connect applies the transformation and then passes the messages to the sink consumer.
+
+After you define a predicate, you can reuse it and apply it to multiple transforms.
+Predicates also include a `negate` option that you can use to invert a predicate so that the predicate condition is applied only to records that do _not_ match the condition that is defined in the predicate statement.
+You can use the `negate` option to pair the predicate with other transforms that are based on negating the condition.
+
+.Predicate elements
+Predicates include the following elements:
+
+* `predicates` prefix
+* Alias (for example, `isOutboxTable`)
+* Type (for example, `org.apache.kafka.connect.transforms.predicates.TopicNameMatches`).
+  Kafka Connect provides a set of default predicate types, which you can supplement by defining your own custom predicates.
+* Condition statement and any additional configuration properties, depending on the type of predicate (for example, a regex naming pattern)
+
+.Default predicate types
+The following predicate types are available by default:
+
+HasHeaderKey:: Specifies a key name in the header in the event message that you want Kafka Connect to evaluate.
+The predicate evaluates to true for any records that include a header key that has the specified name.
+
+RecordIsTombstone:: Matches Kafka _tombstone_ records.
+The predicate evaluates to `true` for any record that has a `null` value.
+Use this predicate in combination with a filter SMT to remove tombstone records.
+This predicate has no configuration parameters.
++
+A tombstone in Kafka is a record that has a key with a 0-byte, `null` payload.
+When a {prodname} connector processes a delete operation in the source database, the connector emits two change events for the delete operation:
+
+* A delete operation (`"op" : "d"`) event that provides the previous value of the database record.
+* A tombstone event that has the same key, but a `null` value.
++
+The tombstone represents a delete marker for the the row.
+When link:{link-kafka-docs}#compaction[log compaction] is enabled for Kafka, the presence of a tombstone causes Kafka to remove all events with the same key when it compacts the topic.
+Even after the earlier messages are removed, the presence of the tombstone event enables consumers to read the topic from the beginning and retrieve the current state of any events.
+Log compaction occurs periodically, with the compaction interval controlled by the link:{link-kafka-docs}#topicconfigs_delete.retention.ms[`delete.retention.ms`] setting for the topic.
++
+In some cases you might want to configure {prodname} so that it does not emit tombstone events.
```

Comment:
I'd slightly reword this:

> While you can configure Debezium so it does not emit tombstone events to begin with (see ...), it can make sense in some cases to filter out tombstones for specific sink connectors, using an SMT and this predicate. That way, they can be used for their original purpose of log compaction in Kafka itself, while being ignored by sink connectors which cannot process them.

(just the gist, I hope it gets clear what I mean)